torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -1368,6 +1368,15 @@ const std::vector<std::string> functions = {
                 return grad_output * mask, None, None
             return torch.threshold(self, threshold, value), backward
 
+        def softplus(self,
+                      beta: number,
+                      threshold: number):
+            result = torch.softplus(self, beta, threshold)
+            def backward(grad_output):
+                z = torch.exp(result * beta)
+                return torch.where( (result * beta) > threshold, grad_output, grad_output * (z - 1.) / z), None, None
```

Comment:
Thanks for going over the implementation!

I wasn't paying enough attention to this in our internal review, I was expecting jit meta testing to catch the error if something was done wrong.
Let me double check the math. :stuck_out_tongue_closed_eyes:

Reply:
Ah, I saw what's going on here.
We are using the output here to compute z; while in the original kernel it's using the input and the math is equivalent.

-> z = torch.exp(result * beta)
In aten, z = std::exp(b * beta), where b is `self` instead of `output`.